src/js/tabs/osd.js

```diff
@@ -817,6 +828,24 @@ OSD.constants = {
       positionable: true,
       preview: FONT.symbol(SYM.ARROW_EAST)
     },
+    LINK_QUALITY: {
+      name: 'LINK_QUALITY',
```

Comment:
There are elements with 4 spaces indentation, and others with 2. Maybe is good to put 4 spaces in at least all the new code?